webapp/views.py

```diff
@@ -943,6 +943,10 @@ def marketo_submit():
     except Exception:
         pass
 
+    enrichment_fields["Google_Consent_Mode__c"] = form_fields.pop(
+        "Google_Consent_Mode__c"
```

Comment:
```suggestion
        "Google_Consent_Mode__c", None
```
This prevents it raising a KeyError if no "Google_Consent_Mode__c" is found